docs/.vuepress/public/.gitkeep

```diff
@@ -0,0 +1 @@
+🏺
```

Comment:
We need the `public` directory when we run VuePress, but git won't let me commit an empty directory. So I'm putting a temporary file `.gitkeep`. The contents of it can be anything we like, but I've chosen the emoji used in ostracon.